api/src/pcapi/core/search/backends/algolia.py

```diff
@@ -760,6 +761,35 @@ def _clean_processing_queue(self, processing_queue: str, originating_queue: str)
                     extra={"queue": originating_queue, "processing_queue": processing_queue},
                 )
 
+    def seach_offers_ids(self, query: str = "", filters: str = "", count: int = 20) -> list[int]:
+        ids: list[int] = []
+        params: dict[str, str | int] = {"filters": filters}
+        for page, _count in enumerate(
+            range(
+                0,
+                count,
+            )
+        ):
+            # handle alogolia pagination of results
+            hits_per_page = min(count, count - _count, 1000)
+            params["page"] = page
+            params["hitsPerPage"] = hits_per_page
+            try:
+                results = self.algolia_offers_client.search(query, params)
+            except Exception as exp:
+                logger.exception("Failed to search in algolia: %s", exp)
```

Comment:
```suggestion
                logger.exception("Failed to search in algolia")
```
pas besoin de rajouter le nom de l'exception dans le message de log, ça va apparaître dans la stacktrace